Source/Core/VideoCommon/GXPipelineTypes.h

```diff
@@ -35,10 +35,18 @@ struct GXPipelineUid
   // and this map lookup can happen every draw call. However, as using memcmp() will also compare
   // any padding bytes, we have to ensure these are zeroed out.
   GXPipelineUid() { std::memset(static_cast<void*>(this), 0, sizeof(*this)); }
+#ifdef _MSC_VER
+#pragma warning(push)
+// Disable warning for uninitialized member variables
```

Comment:
Can you add a reason to this comment, or address it so it isn't necessary?
My guess is that VS doesn't recognize the `memcpy` as initialization?

Reply:
Yep, it's because it doesn't recognize memcpy (but it _does_ recognize memset?)